engine/src/main/java/org/terasology/rendering/dag/stateChanges/BindFBO.java

```diff
@@ -38,6 +54,13 @@ public StateChange getDefaultInstance() {
 
     @Override
     public RenderPipelineTask generateTask() {
-        return null;
+        task = new BindFBOTask(fboId);
+        return task;
```

Comment:
On second thought, and in line with what's written below, if the state change has already generated a task it should:
- resubscribe to the FBM
- call the update() method to update the task's fboId
- return the existing task

So, effectively, the task for a specific FBO is a singleton.